tests/bson/bson-fromPHP_error-005.phpt

```diff
@@ -7,7 +7,7 @@ require_once __DIR__ . '/../utils/tools.php';
 
 class MyRecursiveSerializable implements MongoDB\BSON\Serializable
 {
-    public $x = 1;
+    public $child = 1;
```

Comment:
Based on the message, I assume this change was necessary to demonstrate that BSON serialization never even reaches this property. That would have been difficult before since both properties shared the same name ("x").